### PR TITLE
Add Concur Expense Approvals guide

### PR DESCRIPTION
**Pull request by indrapaul24**

### CODE REVIEW

File: plugins/sap-concur-approve-expense-report/README.md [hunk 1]
```diff
@@ -17,7 +17,689 @@ systems:
...
+
+
+  ![image.png](./architecture.png)
+
+There are four steps to support Concur Expense report approvals using Creator Studio:
+
+1. Setup polling for new approvals (by default, every 60 seconds)
+    - This is required to retrieve newly submitted expense reports
+2. Send a notification to Moveworks with the expense report details and slots to approve or reject the report
+3. Store approval notifications you have already sent in a database queue
```
Comment (ajaymerchia) [inline plugins/sap-concur-approve-expense-report/README.md:181]: database queue -> database

File: plugins/sap-concur-approve-expense-report/README.md [hunk 2]
```diff
@@ -17,7 +17,689 @@ systems:
...
+    "expenseRejectedComment": "Rejecting Report through API"
+}'
+```
+
+## Step 3: Write your Orchestration Code in your Middleware Platform
+
+This diagram describes the flow we will build in this guide.
+
+
+  ![image.png](./architecture.png)
```
Comment (ajaymerchia) [inline plugins/sap-concur-approve-expense-report/README.md:174]: CleanShot 2024-10-31 at 08 17 57@2x

Comment (ajaymerchia) [reply plugins/sap-concur-approve-expense-report/README.md:174]: We can also drop the user identity one

File: plugins/sap-concur-approve-expense-report/README.md [hunk 3]
```diff
@@ -17,7 +17,689 @@ systems:
...
+```bash
+curl --location 'https://us2.api.concursolutions.com/oauth2/v0/token' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'client_id={client_id}' \
+--data-urlencode 'client_secret={client_secret}' \
+--data-urlencode 'refresh_token={refresh_token}' \
+--data-urlencode 'grant_type=refresh_token'
+```
+
+### **API 2: Get All Users**
```
Comment (ajaymerchia) [inline plugins/sap-concur-approve-expense-report/README.md:71]: It doesn't look like this gets used in this guide & we can maybe defer this to Moveworks Setup

File: plugins/sap-concur-approve-expense-report/README.md
Comment (somu-a) [inline plugins/sap-concur-approve-expense-report/README.md:1]: Switch the order of the intro sentence. 

- Loop in the right approvers at the right time to streamline the process and simplify administrative workflows to have expense reports approved or rejected on time in SAP Concur.

New: Have expense reports approved or rejected on time in SAP Concur by looping in the right approvers are the right time to simplify administrative workflows.

**Review by ajaymerchia — APPROVED** [review]

**Pull request merged**